jdat_notebooks/background_estimation_imaging/Imaging Sky Background Estimation.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    from astropy.nddata import NDData
</pre><p>this import does not seem to be used in the notebook (and runs successfully without)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>